src/snowflake/snowpark/mock/_connection.py

```diff
@@ -662,7 +662,6 @@ def get_result_query_id(self, plan: SnowflakePlan, **kwargs) -> str:
         result_set, _ = self.get_result_set(plan, to_iter=True, **kwargs)
         return result_set["sfqid"]
 
-    
```

Comment:
accident?

Reply:
I think this was a side effect of running the precommit fix_lint